### PR TITLE
selinux: Fix setroubleshoot listing behavior

### PR DESCRIPTION
**Pull request by dperpeet**
This needed to reflect the latest listing pattern changes.

@andreasn This deviates from the original design since I decided to keep the occurrence count as a badge in expanded mode so that the line doesn't change when expanding.

### CODE REVIEW

**Comment by larsu** [comment]
Thanks, this is much better than before!

The title "SELinux Access Control errors" is set in a much heavier font than the titles in the design document.

I know there was some discussion around when to expand the row. I didn't follow the outcome, but highlighting the row on hover signifies to me that something will happen when I click it. However, it only expands and collapses when clicking the arrow right now.

**Comment by dperpeet** [comment]
tests need fixing

File: pkg/base1/cockpit.css [hunk 1]
```diff
@@ -665,6 +665,7 @@ tr.listing-item .listing-toggle {
     padding: 0 !important;
     width: 35px;
     color: #000000;
+    cursor: pointer;
```
Comment (dperpeet) [inline pkg/base1/cockpit.css]: @stefwalter @larsu I think this is a valid addition to the base stylesheet: the toggle element should always be clickable

**Comment by dperpeet** [comment]
I addressed the issues and adapted the tests. The listing doesn't have to be 100% optimized, since it'll be replaced by the standard listing pattern soon anyway.

**Comment by dperpeet** [comment]
after discussing this with @larsu I need to change the highlight/click behavior:
- the entire row should catch a user click
- if navigate to item is disabled (as it is in this case) then only the caret should be highlighted

**Pull request closed without merging**